src/engraving/libmscore/ornament.h

```diff
@@ -0,0 +1,74 @@
+#ifndef ORNAMENT_H

```

Comment:
You missed the license for both .h/.cpp files

Also, this define should be MU_ENGRAVING_ORNAMENT_H